test/integrations/destinations/tiktok_ads/processor/data.ts

```diff
@@ -6870,4 +6870,107 @@ export const data = [
       },
     },
   },
+  {
```

Comment:
@anantjain45823  do you have test case with no contents but products ?

Reply:
Yes, we have it. [Ref](https://github.com/rudderlabs/rudder-transformer/blob/bf506ea1505bbaf220ad91efc329986a98b32643/test/integrations/destinations/tiktok_ads/processor/data.ts#L6349)
Since gaurav made the latest changes so he can confirm if some other test case on products is required or not